Worksheet_B/README.MD

```diff
@@ -0,0 +1,12 @@
+```
+Player Starts Minigame
+A five Letter word is randomly generated
+Player has 4 tries to guess the correct code
+game ends after 4 incorrect tries or the correct guess
+if the player guesses correctly then open door or chest
+if generated word is "grass" and player guesses "glass" remove a try and show a likeness of 4
+The likeness is how many letters are in both the players guess and the generated word
```

Comment:
The worksheet asked you to provide pseudocode for calculating the likeness score, e.g. something like
```
let likeness = 0
for i = 1, ..., wordLength
  if (ith letter of secretWord) == (ith letter of guessedWord)
    increase likeness by 1
  end if
end for
```